templates/opening-hours-widget.html.twig

```diff
@@ -14,10 +14,21 @@
  * @ingroup themeable
  */
 #}
-<div
-  class="openinghours openinghours--{{ widget_type }} openinghours-widget"
-  data-service="{{ service_id }}"
-  data-channel="{{ channel_id }}"
-  data-type="{{ widget_type }}"
-  {% if date %} data-date="{{ date }}"{% endif %}
-></div>
+<div class="openinghours-wrapper">
```

Comment:
This will always trigger 2 requests to the OpeningHours backend. Even if the alternative view is never shown.

Better to show only one widget and switch the type + reload the content when the toggle button is clicked?